packages/core/src/Account.ts

```diff
@@ -368,7 +397,11 @@ export class Account<T = any> extends EventEmitter {
         }
 
         this.logger.log('Last client was permanent - Deleting cryptography stores');
-        await this.service!.cryptography.deleteCryptographyStores();
+        if (this.cryptoProtocolConfig?.proteus) {
+          await this.logout(true);
+        } else {
+          await this.service!.cryptography.deleteCryptographyStores();
+        }
```

Comment:
can't we always use `logout(true)` here and remove the condition? I guess `logout(true)` will take care of deleting the DB